Renes gimmicks: initial upload

PatrykZdziech opened the pull request:
Added 4 working spells
May require balance changes or grafics update.

Renesans123 committed:
Renes gimmicks: initial upload
Added 4 working spells and 2 spell concepts currently suspended
May require balance changes or grafics update.

Renesans123 committed:
Renes Gimmicks: Removed unsupported spells

jdoleary commented:
Hey! Awesome to see another mod in the works!  Ping me when you're totally finished with it and I'll merge it in and ship it with the next update

Renesans123 committed:
Renes Gimmicks:
Reintroduced Caltrops
Updated spellicons with background pattern
Addeed particle effect to Burning Rage
Fixed small issues like desciption and skyBeam

Renesans123 committed:
Renes Gimmicks: Added Vampire bite animation Changed SummonTrap spellicon

jordan-octavia-blue committed:
src: Add Renes Gimmicks to mod list

jordan-octavia-blue commented on Renes_gimmicks/cards/Burning_rage.ts:
```diff
@@ -0,0 +1,179 @@
+                }
+                for (let unit of targets) {
+                    Unit.addModifier(unit, card.id, underworld, prediction, quantity);
+                    unit.damage += quantity * attackMultiplier;
```
Oh! It increases the unit's damage!  Very cool, I think the spell description could be a little more clear on that.  Maybe "but also increases the target's damage by ${attackMultiplier}"

jordan-octavia-blue commented on Renes_gimmicks/cards/Burning_rage.ts:
```diff
@@ -0,0 +1,179 @@
+                Unit.takeDamage(unit, modifier.quantity * damageMultiplier, undefined, underworld, prediction);
+                FloatingText.default({
+                    coords: unit, 
+                    text: `${Math.pow(modifier.quantity, 2)} rage damage`,
```
I believe this should be identical to the damage amount in Unit.takeDamage above so that the floating text shows how much damage they actually take.

Renesans123 committed:
Renes gimmicks: responce to review
Fixed damage displayed in Burnig rage
changed 2 spell descriptions

jordan-octavia-blue committed:
Burning_rage: Remove particles when modifier removed
